__tests__/app.test.js

```diff
@@ -130,6 +130,38 @@ describe('/api/articles', () => {
             expect(response.statusCode).toBe(404);
         })
     });
+    test('GET: 200 responds with array containing article objects filtered by topic query', () => {
+        const topic = 'mitch'
+        return request(app)
+        .get(`/api/articles?topic=${topic}`)
+        .expect(200)
+        .then((response) => {
+            const articles = response.body;
+            expect(articles).toHaveLength(12);
+            articles.forEach((article) => {
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    topic: 'mitch',
+                    comment_count: expect.any(Number)
+                });
+            })
```

Comment:
Excellent testing, good use of the exact value 'mitch' for topics, while leaving everything else open ended